src/renderers/shared/event/EventPluginRegistry.js

```diff
@@ -128,6 +128,12 @@ function publishRegistrationName(registrationName, PluginModule, eventName) {
   EventPluginRegistry.registrationNameModules[registrationName] = PluginModule;
   EventPluginRegistry.registrationNameDependencies[registrationName] =
     PluginModule.eventTypes[eventName].dependencies;
+
+  if (__DEV__) {
+    var lowerCasedName = registrationName.toLowerCase();
+    EventPluginRegistry.getPossibleRegistrationName[lowerCasedName] =
```

Comment:
`possibleRegistrationNames` is a little shorter and aligns better with the naming of the surrounding code – can we go with that?